src/scripts/modules/media/body/content-baked.less

```diff
@@ -220,12 +220,28 @@ h1.example-title .text {
   }
 }
 
+
+//
+// MathJax_Display override
+// --------------------------------------------------
+.MathJax_Display {
+  width: auto !important;
```

Comment:
Can you include a description of why the override occurs? Also, so that it shows up in dev tools, I think the following format works: `width: auto !important /*because ....*/;`

Reply:
I'm not sure if I understand you correct. That's the only possible way I've found to override default MathJax styles.

I also thought of putting all mathjax elements into some containers and style the container + reset all mathjax displaying properties (not the math elements structure itself) but someone said that's not the best idea.

I think we should be able to style MathJax elements through the MathJax config file, am I right?

Which of those might be the best choice?